core/translate.py

```diff
@@ -121,14 +122,12 @@ def _get_json5_from_google(self, text):
         return json
 
     def _get_translation_from_json5(self, content):
-        result = ""
-        pos = 2
-        while True:
-            m = self.match_string.match(content.decode('utf-8'), pos)
-            if not m:
-                break
-            result += m.group(1)
-            pos = m.end()
+        #print(content.decode('utf-8'))
+        response = content.decode('utf-8')
+        fixedJSON = re.sub(r',{2,}', ',', response).replace(',]', ']')
+        data = json.loads(fixedJSON)
+        #print(json.dumps(data, sort_keys=False, indent=2, separators=(',', ': ')))
+        result = data[0][0][0]
```

Comment:
From what I've seen, the returned array is more or less:
```
[
  [
    ["sentence 1",
    ...],
    ["sentence 2",
    ...]
  ],
  more data,
  ...
]

```
data[0] consists of one array per sentence, with index 0 of each array containing the translated string. So, one possible solution is:

`result = "".join([x[0] for x in data[0]])`